README.md

```diff
@@ -220,6 +220,144 @@ C:\Users\bion\Desktop>dumpbin /IMPORTS .\example.exe | findstr msvcp
+`lit` invocations, run the tests with `ctest -V`.
+
+## Running A Subset Of The Tests
+
+`${PROJECT_BINARY_DIR}\tests\llvm-lit\llvm-lit.py` can be invoked on a subdirectory of a testsuite and will execute all
+the tests under that subdirectory. This can mean executing the entirety of a single testsuite, running all tests under
+a category in libcxx, or running a single test in `std` and `tr1`.
+
+## Examples
+
+```
+:: This command configures the project with tests enabled. It assumes you are using the vcpkg submodule and have
+:: already installed boost. It also assumes you have inited and updated the llvm-project submodule.
+C:\STL\build>cmake -GNinja -DCMAKE_CXX_COMPILER=cl -DCMAKE_TOOLCHAIN_FILE=..\vcpkg\scripts\buildsystems\vcpkg.cmake^
+ -DENABLE_TESTS=TRUE ..
+
+:: As stated above this step is only strictly necessary if you have yet to build the STL or if you have changed the
+:: output directory of the binaries. Any changes or additions in any of the existing testsuites do not require
+:: recompilation or reconfiguration to take effect when running the tests.
+C:\STL\build>ninja
+
+:: This command will run all of the testsuites with verbose output.
+C:\STL\build>ctest -V
+
+:: This command will run all of the tr1 testuite
+C:\STL\build>ctest -R tr1
+
+:: This command will also run all of the tr1 testuite.
+C:\STL\build>python tests\llvm-lit\llvm-lit.py ..\tests\tr1
+
+:: If you want to run a subset of a testsuite you need to point it to the right place in the sources. The following
+:: will run the single test found under VSO_0000000_any_calling_conventions.
+C:\STL\build>python tests\llvm-lit\llvm-lit.py ..\tests\std\tests\VSO_0000000_any_calling_conventions
+
+:: You can invoke llvm-lit with any arbitrary subdirectory of a testuite. In libcxx this allows you to have a finer
+:: control over what category of tests you would like to run. The following will run all the libcxx map tests.
+C:\STL\build>python tests\llvm-lit\llvm-lit.py ..\llvm-project\libcxx\test\std\containers\associative\map
+```
+
+## Interpreting The Results Of Tests
+
+## CTest
+
+When running the tests via CTest each of the testsuites is considered to be a single test. If any single test in a
+testsuite fails, CTest will report the test which represents that testsuite as failed.
+
+Example:
+```
+67% tests passed, 1 tests failed out of 3
+
+Total Test time (real) = 2441.55 sec
+
+The following tests FAILED:
+      1 - libcxx (Failed)
+```
+
+CTest will output everything that was sent to stderr for each of the failed testsuites, which can be used to identify
+which individual test within the testsuite failed. It can sometimes be helpful to run CTest with the `-V` option in
+order to see the stdout of the tests.
+
+## llvm-lit
+
+When running the tests directly via the generated `llvm-lit.py` script the result of each test will be printed. The
+format of each result is {[Result Code](#result-code-values)}: {Testsuite Name} :: {Test Name}:{Configuration Number}.

```

Comment:
```suggestion
format of each result is `{[Result Code](#result-code-values)}: {Testsuite Name} :: {Test Name}:{Configuration Number}`.
```

Reply:
I tried that, you can't have links inside of a code snipet.

Reply:
Maybe I should drop the link?